libcaja-private/caja-file.c

```diff
@@ -4866,9 +4866,15 @@ caja_file_fit_date_as_string (CajaFile *file,
 	}
 
 	today = g_date_time_new_now_local ();
-	file_date_age = g_date_time_difference (today, date_time);
+	end_of_today = g_date_time_add_full(today, 0, 0, 1,
+			-1 * g_date_time_get_hour(today),
+			-1 * g_date_time_get_minute(today),
+			-1.0 * g_date_time_get_seconds(today));
 	g_date_time_unref (today);
```

Comment:
Formatting issue. Can you start line 4870-4872 after the openning bracket from line 4869
Also a blank before brackets are missing missing.

Reply:
I just fixed the formatting as requested and replaced the change commit with a fixed version.

Reply:
Really?
![Screenshot 2023-05-05 at 18-05-25 caja-file fix yesterday_today informal date bug by basicmaster · Pull Request #1710 · mate-desktop_caja](https://user-images.githubusercontent.com/961604/236510173-eee5a16b-c371-4dc4-bc22-cc7901d8a530.png)

I do not get why people do have so much problems to work with tabs.
Personal i don't use tabs in code for myself....so i never run in that problems :-)

Reply:
Usually I try to align to the existing code as much as possible. As the code already uses tabs, I also used tabs for indentation. 

I increased the indent of lines 4870 to 4872 so that the text start is horizontally to the right of the opening bracket in line 4869....so what I am still missing here? :-)

Reply:
Perfect , Danke schön :)